apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts

```diff
@@ -180,7 +185,8 @@ export class BulkScriptAction extends BaseScriptAction {
       parameterLongName: '--from',
       parameterShortName: '-f',
       argumentName: 'PROJECT2',
-      description: 'Run command in all projects that directly or indirectly depend on the specified project'
+      description: 'Run command in all projects that directly or indirectly depend on the specified project. ' +
```

Comment:
Nit: 1) There are two spaces after the period here, but not in the above change.
2) Can the wording be a bit better? Something like "use . to specify the project in the pwd.".

Reply:
1) Oops. Copy+paste error.
2) Sure. Rephrased a bit.